sourcecode/proxies/admin/Dockerfile

```diff
@@ -18,6 +18,7 @@ CMD bash /createEnvFile.sh /usr/share/nginx/html; nginx -g "daemon off;"
 
 FROM node:16.13 as dev
 WORKDIR /app
+RUN apk --no-cache add curl
```

Comment:
The image needs to be changed to Alpine, or this won't work. `curl` is already present in the Debian-based image.

Reply:
Updated